style.css

```diff
@@ -6,114 +6,424 @@
 
 body {
   font-family: 'Poppins', sans-serif;
+  background: #e5e5e5;
   margin: 0;
 }
 
-.container-parent {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-}
+@media (max-width: 768px) and (min-width: 375px) {
+  .container-parent {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
 
-.top-navigation {
-  background-color: white;
-  position: fixed;
-  width: 100vw;
-  top: 0;
-  left: 0;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-}
+  .top-navigation {
+    background-color: white;
+    position: fixed;
+    width: 100vw;
+    top: 0;
+    left: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    z-index: 5;
+  }
 
-.container-logo {
-  width: 36.66%;
-  padding: 1.66% 2.2%;
-  margin-top: 1.07%;
-  margin-left: 3.2%;
-  display: flex;
-  align-items: center;
-}
+  .container-logo {
```

Comment:
- [ ] As we are imitating the Figma design while comparing your design to the original one, I can see that your the background is not as well set as the one for figma, please improve your background so it looks more appealing and professional. :wink: 

Your Design 
![Screenshot from 2023-03-26 00-42-18](https://user-images.githubusercontent.com/88894525/227745861-18b384f3-8dbc-4bad-af82-c5be4c3c5acc.png)

Figma Design 
![Screenshot from 2023-03-25 17-46-06](https://user-images.githubusercontent.com/88894525/227745875-b154c3e9-7204-4280-ae99-26ac6c73cf3a.png)

Reply:
Hi, just a quick question here. Aren't I suppose to get approval for the: 4 card sections and the about me section only here for this particular pull request?